src/main/java/com/team4/testingsystem/dto/QuestionDTO.java

```diff
@@ -0,0 +1,91 @@
+package com.team4.testingsystem.dto;
+
+import com.team4.testingsystem.entities.Level;
+import com.team4.testingsystem.entities.Module;
+import com.team4.testingsystem.entities.User;
+
+public class QuestionDTO {
```

Comment:
Here, DTO means an object we are going to use for communicating with frontend. Don't think they should use ID's for level or module. I would expect them to specify them by its names

Reply:
do i need to replace user id with user name too?

Reply:
Yes